### ISSUE TITLE
Help required to fix 'must be Input Type but got' error

### ISSUE THREAD

**Issue by renepardon** [cjoudrey/graphql-schema-linter#156]
Hello,

I tried your validator and got the same errors like in this question: https://spectrum.chat/thread/9ee3d211-566d-4250-849c-2f8e4b42268c

**Error*
```bash
$ graphql-schema-linter schema.graphql 
It looks like you may have hit a bug in graphql-schema-linter.

It would be super helpful if you could report this here: https://github.com/cjoudrey/graphql-schema-linter/issues/new

Error: The type of AddPetInput.pet must be Input Type but got: Pet.

The type of UpdatePetInput.pet must be Input Type but got: Pet.

The type of PlaceOrderInput.order must be Input Type but got: Order.

The type of CreateUserInput.user must be Input Type but got: User.

The type of CreateUsersWithArrayInputInput.user must be Input Type but got: [User].

The type of CreateUsersWithListInputInput.user must be Input Type but got: [User].

The type of UpdateUserInput.user must be Input Type but got: User.
    at assertValidSchema (/usr/local/lib/node_modules/graphql-schema-linter/node_modules/graphql/type/validate.js:82:11)
```

**To reproduce this:**

```
git clone git@github.com:renepardon/openapi-generator.git /tmp/openapi-generator ; git checkout graphql_client ; cd /tmp/openapi-generator/samples/server/petstore/graphql-server/openapi3graphql-server ; npm i ; npm run start
```
